migrate.go

```diff
@@ -218,8 +232,8 @@ func createVersionTable(db *sql.DB) error {
 
 // wrapper for EnsureDBVersion for callers that don't already have
 // their own DB instance
-func GetDBVersion(db *sql.DB) (int64, error) {
-	version, err := EnsureDBVersion(db)
+func (c *Client) GetDBVersion(db *sql.DB) (int64, error) {
```

Comment:
I searched around to try to find some reason this exists, and didn't find any. Considering your agreement with that, it seems worth killing.